### PR TITLE
[Rooms] QuickStart project improvements

### PR DESCRIPTION
**Pull request by Mrayyan**
## Purpose

Improvements based on team feedback for project

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: rooms-quickstart-java/src/main/java/com/communication/rooms/quickstart/App.java [hunk 1]
```diff
@@ -277,34 +279,36 @@ public static void main(String[] args) {
...
                         String roomId = br.readLine();
-                        
+
                         listParticipants(roomId);
 
                         break;
                     }
                     case 10:
-                        System.out.println("Deleting all rooms");
+                        System.out.println("Deleting all rooms and existing program...");
```
Comment (mikehang-msft) [inline rooms-quickstart-java/src/main/java/com/communication/rooms/quickstart/App.java:304]: nit: exiting

File: rooms-quickstart-java/src/main/java/com/communication/rooms/quickstart/App.java [hunk 2]
```diff
@@ -134,16 +130,22 @@ public static void updateRoom(String roomId) {
...
+                    if (count >= 3) {
+                        break;
+                    }
+                }
 
-            for (CommunicationRoom room : rooms) {
-                System.out.println("\nFirst two room ID's in the list of rooms: " + room.getRoomId());
-                count++;
-                if (count >= 2) {
+                if (count >= 3) {
```
Comment (mikehang-msft) [inline rooms-quickstart-java/src/main/java/com/communication/rooms/quickstart/App.java:148]: you can remove those lines

File: rooms-quickstart-java/src/main/java/com/communication/rooms/quickstart/App.java (hunk 1 above)
Comment (Mrayyan) [reply rooms-quickstart-java/src/main/java/com/communication/rooms/quickstart/App.java:304]: fixed

File: rooms-quickstart-java/src/main/java/com/communication/rooms/quickstart/App.java (hunk 2 above)
Comment (mikehang-msft) [reply rooms-quickstart-java/src/main/java/com/communication/rooms/quickstart/App.java:148]: I think you are counting on rooms. Let print all rooms in the first 2 pages and break out of the loop. You do not need to check inside loop anymore. The inner loop then can be simplified as below.




collection.getElements().forEach(room-> System.out.println(room));

**Review by mikehang-msft — APPROVED** [review]

**Pull request merged**